### DISCUSSION TITLE
[Bug] Content Field of type Select: Labels missing

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#272] [Help] [answered]
Hello,
I'm experiencing a buggy behaviour when trying to add a Content Field of type Select to the Products model. 
When I first add the field, I'm met with this UI:
<img width="696" alt="Screenshot 2023-06-14 at 14 24 59" src="https://github.com/swellstores/community/assets/94485687/68c35363-26c1-4ed3-929e-524b2d25144d"> 
As you can see there is room to enter a "Label" and "Value" for each field. But I noticed that after doing that, the dropdown on the product displays the values and not the labels. Also, when I tried to edit this field again on the model, the UI had changed, now missing the labels:
<img width="682" alt="Screenshot 2023-06-14 at 14 25 29" src="https://github.com/swellstores/community/assets/94485687/f781c2b8-7605-4ca9-90c5-cfa4514036ce">

This behaviour happens every time I try to add a field of type "select" to the Products model. Also, I'm selecting "Details" as Admin field location and "Content" as API namespace.

**Comment by igoraugustynski** [comment]
Anyone?

**Reply by sags95** [reply, marked as answer]
@igoraugustynski I've checked with the team and this is a bug that has already been logged. I've bumped the priority and it's looking like a quick fix.

**Reply by steven-k1** [reply]
@igoraugustynski A fix has been pushed to production for this.

**Reply by igoraugustynski** [reply]
Actually it is still somewhat broken. In the table that is displayed the fields contain the value instead of the label.

**Reply by igoraugustynski** [reply]
@steven-k1 I have also noticed some very weird behaviour when adding items to a bundle -> The autocomplete seems completely broken. Can You guys check this?

**Reply by igoraugustynski** [reply]
@steven-k1 @sags95 Here is a screenshot of what's happening. Looks like the autocomplete component correctly makes the API call to search using the string i input, but fails to displays the results. 
<img width="2682" alt="Screenshot 2023-08-08 at 19 26 19" src="https://github.com/swellstores/community/assets/94485687/9e1fc551-b368-40ee-9f5e-c27b8d94a4a6">

**Reply by igoraugustynski** [reply]
Anyone? This makes creating bundles almost impossible..

**Reply by ericingram** [reply]
👀

**Reply by steven-k1** [reply]
@igoraugustynski Looking into these additional issues and will get back to you once I have further details.

**Comment by steven-k1** [comment]
@igoraugustynski  Circling back to the continued issue you mentioned:

> Actually it is still somewhat broken. In the table that is displayed the fields contain the value instead of the label.

Is this in reference to the fields within the admin models section here:

![image](https://github.com/swellstores/community/assets/77881204/a4788b91-62cb-4598-a7e3-1e4c76a6ba74)

Or are you referring to the fields within the product display here:

![image](https://github.com/swellstores/community/assets/77881204/a1d6d28b-9d48-4aa7-a13e-ff8a88b58581)

Additionally, for what you mentioned here:

> @steven-k1 @sags95 Here is a screenshot of what's happening. Looks like the autocomplete component correctly makes the API call to search using the string i input, but fails to displays the results. <img alt="Screenshot 2023-08-08 at 19 26 19" width="2000" src="https://user-images.githubusercontent.com/94485687/259179839-9e1fc551-b368-40ee-9f5e-c27b8d94a4a6.png">

Are you still seeing this issue on your end? I tested and saw that the results were displaying.

**Reply by igoraugustynski** [reply]
1. There were two fields before, "label" and "value". Now it's only "value" so I guess it's okay now. 
2. Seems to be working correctly!

**Reply by steven-k1** [reply]
@igoraugustynski Thanks for confirming issue two is resolved.

As for the label and value fields, I'll look into this to see if we made any changes on our end and get back to you once I have more details.
